tests/countries/test_france.py

```
@@ -29,176 +33,367 @@ def setUp(self):
+            ("2022-05-01", "Fête du Travail"),
+            ("2022-05-08", "Fête de la Victoire"),
+            ("2022-05-26", "Ascension"),
+            ("2022-06-06", "Lundi de Pentecôte"),
+            ("2022-07-14", "Fête nationale"),
+            ("2022-08-15", "Assomption"),
+            ("2022-11-01", "Toussaint"),
+            ("2022-11-11", "Armistice"),
+            ("2022-12-25", "Noël"),
+        )
+
+    def test_subdiv_deprecation(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)